python/lsst/analysis/tools/actions/plot/scatterplotWithTwoHists.py

```diff
@@ -711,17 +711,17 @@ def _makeSideHistogram(
     ) -> None:
         # Side histogram
         sideHist = figure.add_subplot(gs[1:, -1], sharey=ax)
```

Comment:
Pre-existing, but there is a rogue `# Side histogram` comment at line 701.

Reply:
This comment is just reminding people that they are in the chunk that defines the side histogram.